index.html

```diff
@@ -244,25 +234,41 @@ <h2 class="contact-me-header">
           chat please get <br />
           in touch.
         </h2>
-        <div class="input-form">
-          <input type="text" placeholder="Full name" /><input
+        <form
+          autocomplete="off"
+          action="https://formspree.io/f/xdovwjar"
+          method="post"
+          class="input-form"
+        >
+          <input
+            co
+            type="text"
+            placeholder="Full name"
+            maxlength="30"
+            name="full-name"
+            required
+          />
+          <input
+            required
             type="email"
             placeholder="Email address"
+            name="email"
           />
           <textarea
             class="details"
             id="details"
             cols="20"
             rows="30"
             placeholder="Hey I’d like to connect to..."
+            maxlength="500"
+            required
+            name="message"
           ></textarea>
-        </div>
-        <div class="green-btn">
-          <a href="https://github.com/GeekyHacks">
-            <button type="button" title="press">Get in touch</button>
-          </a>
-        </div>
-        <div class="contact-email">geekyhacks22@gmail.com</div>
+          <button name="submit" class="green-btn" type="submit" title="press">
```

Comment:
Please follow the design guideline and add the Hover and Active state styling for your form submit button. 👍 The main goal of this project is to code the given design perfectly.  
<img width="570" alt="Screenshot 2023-04-04 at 9 39 25 PM" src="https://user-images.githubusercontent.com/25887253/229887842-b094cf91-f9e5-404d-805a-3d0260efccfd.png">